scripts/upload-source-maps.js

```diff
@@ -43,14 +44,18 @@ async function uploadSourceMaps(site, apiKey, packageName, bundleFolder) {
 }
 
 async function main() {
-  for (const packageName of ['logs', 'rum', 'rum-slim']) {
+  for (const { packageName, service } of [
+    { packageName: 'logs', service: 'browser-logs-sdk' },
+    { packageName: 'rum', service: 'browser-rum-sdk' },
+    { packageName: 'rum-slim', service: 'browser-rum-sdk' },
+  ]) {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ ‏what about extracting a constant for that configuration?